worlds/pokemon_rb/locations.py

```diff
@@ -0,0 +1,1722 @@
+
+from BaseClasses import Location
+from .rom_addresses import rom_addresses
+loc_id_start = 17200000
+locations = []
```

Comment:
this global "locations" doesn't seem to be used, just filled, but also probably shouldn't be used. It seems to keep filling up and up with every generation until it might exhaust the webhost's memory.